src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/DocumentServiceTest.kt

```diff
@@ -358,7 +360,7 @@ internal class DocumentServiceTest {
   fun `update a document category to a versioned category also sets version to 1`() {
     val documentId = ::DocumentId.random()
     val now = OffsetDateTime.now()
-    val document = Document(documentId, recallId, UNCATEGORISED, "parta.pdf", null, null, now, createdByUserId)
+    val document = Document(documentId, recallId, UNCATEGORISED, FileName("parta.pdf"), null, null, now, createdByUserId)
```

Comment:
Same thing - albeit was present before - and yes, makes no difference in practice.